source/includes/options-mongosqld.yaml

```diff
@@ -68,6 +68,12 @@ description: |
 
   To enable TLS/SSL in this connection, use the :option:`--mongo-ssl`
   option.
+
+  The {{role}} option also supports the :urioption:`readPreference` and
+  :urioption:`readPreferenceTags` connection string URI Options. To
```

Comment:
"Options" shouldn't be upper-case.

Consider linking to `mongodb-uri` here as well, on the principle of "one link per paragraph".